### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by kgaillot**
This will be backported to 2.1

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this when you get a chance? Thanks

File: tools/crmadmin.c [hunk 1]
```diff
@@ -131,17 +133,18 @@ static pcmk__supported_format_t formats[] = {
...
 build_arg_context(pcmk__common_args_t *args, GOptionGroup **group) {
     GOptionContext *context = NULL;
 
-    const char *description = "Notes:\n\n"
-                              "Time Specification:\n\n"
-                              "The TIMESPEC in any command line option can be specified in many different\n"
-                              "formats.  It can be just an integer number of seconds, a number plus units\n"
-                              "(ms/msec/us/usec/s/sec/m/min/h/hr), or an ISO 8601 period specification.\n\n"
-                              "Report bugs to " PCMK__BUG_URL;
-
```
Comment (clumens) [inline tools/crmadmin.c:144]: Also add the following:
```
--- a/tools/crmadmin.8.inc
+++ b/tools/crmadmin.8.inc
@@ -3,3 +3,6 @@ crmadmin [options] [node]
 
 /the Pacemaker controller/
 .SH OPTIONS
+
+/query information/
+.SH NOTES
```

Comment (kgaillot) [reply tools/crmadmin.c:144]: thanks!

Comment (kgaillot) [reply tools/crmadmin.c:144]: updated commit

**Pull request merged**